README.md

```diff
@@ -183,6 +183,8 @@ You can skip configuring multipath, if you choose not to enable mpath.
 Below we set mapth in Active/Passive mode; Note currently Active/Active is not supported.
 # modprobe dm_multipath
 # mpathconf --enable
+After tha cat command, copy paste the below lines in the terminal till Ctrl+C, note that #LIO iSCSI
```

Comment:
s/tha/the/

If you feel cat and ctrl^C command are leading to confusion,

how about just say:

```
Please add the below configuration to /etc/multipath.conf

# LIO iSCSI
 devices {
         device {
                 vendor "LIO-ORG"
                 user_friendly_names "yes" # names like mpatha
                 path_grouping_policy "failover" # one path per group
                 path_selector "round-robin 0"
                 failback immediate
                 path_checker "tur"
                 prio "const"
                 no_path_retry 120
                 rr_weight "uniform"
         }
 }
```

Reply:
Yes, this looks much cleaner.

Reply:
betterment:
Please add the below configuration at the end of /etc/multipath.conf